src/sentry/utils/urls.py

```diff
@@ -28,3 +28,11 @@ def non_standard_url_join(base, to_join):
             return original_base_scheme + match.group(2)
 
     return rv
+
+
+def add_params_to_url(url, params):
+    url_parts = list(urlparse(url))
+    query = dict(parse_qsl(url_parts[4]))
+    query.update(params)
+    url_parts[4] = urlencode(query)
```

Comment:
The `url_parts[4]` part here seems very confusing to me, as it is hard for someone to learn what the fourth element means without reading the docs. I thought urllib.urlparse returns a `named tuple`? Is it possible to directly access the `query` segment of the url by using `url_parts.query`?

Reply:
@Neo-Zhixing I just copied from [here](https://github.com/getsentry/getsentry/blob/master/getsentry/utils/urls.py#L4) but I agree it's confusing.